tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -144,4 +156,207 @@ TEST_CASE("pa_coeff")
    }
 }
 
+TEST_CASE("Hcurl pa_coeff")
+{
+   for (dimension = 2; dimension < 4; ++dimension)
+   {
+      for (int coeffType = 0; coeffType < 2; ++coeffType)
+      {
+         for (int integrator = 0; integrator < 2; ++integrator)
+         {
+            const int ne = 2;
+            std::cout << "Testing " << dimension << "D ND partial assembly with "
+                      << "coeffType " << coeffType << " and "
+                      << "integrator " << integrator << std::endl;
+            for (int order = 1; order < 5; ++order)
+            {
+               Mesh* mesh;
+               if (dimension == 2)
+               {
+                  mesh = new Mesh(ne, ne, Element::QUADRILATERAL, 1, 1.0, 1.0);
+               }
+               else
+               {
+                  mesh = new Mesh(ne, ne, ne, Element::HEXAHEDRON, 1, 1.0, 1.0,
+                                  1.0);
+               }
+               FiniteElementCollection* ND_fec =
+                  new ND_FECollection(order, dimension);
+               FiniteElementSpace ND_fespace(mesh, ND_fec);
+               Array<int> ess_tdof_list;
+
+               BilinearForm paform(&ND_fespace);
+               Coefficient* coeff = nullptr;
+               Coefficient* curlCoeff = nullptr;
+               if (coeffType == 0)
+               {
+                  coeff = new ConstantCoefficient(12.34);
+                  curlCoeff = new ConstantCoefficient(12.34);
+               }
+               else if (coeffType == 1)
+               {
+                  coeff = new FunctionCoefficient(&coeffFunction);
+                  curlCoeff = new FunctionCoefficient(&linearFunction);
+               }
+               paform.SetAssemblyLevel(AssemblyLevel::PARTIAL);
+               if (integrator < 2)
```

Comment:
Isn't this always true?

Reply:
Good catch. I copied this without thinking, from the H1 version, which is in the master branch. Looping over integrator from 0 to 2 is a better test, and I will make that change. This should also be done in master.